project/ws/viewer/src/lib/plugins/html/html.component.ts

```diff
@@ -358,26 +358,30 @@ export class HtmlComponent implements OnInit, OnChanges, OnDestroy {
         //     `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
         //   )
         // }
-        if (this.htmlContent.streamingUrl && this.htmlContent.initFile) {
-          this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(
-            // tslint:disable-next-line:max-line-length
-            `${this.generateUrl(this.htmlContent.streamingUrl)}/${this.htmlContent.initFile}?timestamp='${new Date().getTime()}`
-          )
+        if (this.htmlContent.streamingUrl.includes(environment.azureHost)) {
+          this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(this.htmlContent.streamingUrl)
         } else {
-          if (environment.production) {
+          if (this.htmlContent.streamingUrl && this.htmlContent.initFile) {
```

Comment:
put additional check for this.htmlContent

Reply:
ok adding on top